test/test_helper.exs

```diff
@@ -19,6 +22,7 @@ alias Instream.TestHelpers
   TestHelpers.Connection.child_spec,
   TestHelpers.ConnectionWithOpts.child_spec,
   TestHelpers.JSONConnection.child_spec,
+  TestHelpers.LogConnection.child_spec,
```

Comment:
The child specs for the supervision trees are generated by `Instream.Pool.Spec.spec/1`. It receives the connection module and, using `Connection.config/0` generates the arguments passed to `:poolboy.child_spec/3`.